lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -16,7 +16,24 @@ const defaultConfig = {
   navigations: [
     {
       id: 'default',
-      artifacts: ['Accessibility', 'ConsoleMessages'],
+      artifacts: [
+        'Accessibility',
+        'Appcache',
```

Comment:
they need to be defined in `artifacts` too, once that happens, the snapshot assertion for number of audits we compute should fail :)

Reply:
ah ok.

btw i found this vscode plugin, super useful https://marketplace.visualstudio.com/items?itemName=wmaurer.change-case